docs/vcl-configuration.md

```diff
@@ -207,7 +209,7 @@ No VCL named v-20861922-1561381196 known.
 Command failed with error code 106
 	{"varnish_controller_version": "0.21.0", "varnish_service": "my-varnish", "pod_name": "my-varnish-varnish-0", "namespace": "my-varnish"}
 ```
-
+#TODO document not ready addresses behavior. Also propably add probes to examples as well
```

Comment:
Were you going to do this in this PR or another?

Reply:
oh, I've already documented it in this PR, but forgot to remove the TODO